source/code/source/playfab/PlayFabEventApi.cpp

```diff
@@ -18,22 +18,22 @@ namespace PlayFab
 
     void PlayFabEventAPI::EmitEvent(std::unique_ptr<const IPlayFabEvent> event, const PlayFabEmitEventCallback callback) const
     {
-        auto eventRequest = std::shared_ptr<PlayFabEmitEventRequest>(new PlayFabEmitEventRequest());
+        auto eventRequest = std::unique_ptr<PlayFabEmitEventRequest>(new PlayFabEmitEventRequest());
```

Comment:
>std::unique_ptr<PlayFabEmitEventRequest>(new PlayFabEmitEventRequest()); [](start = 28, length = 72)

recommend std::make_unique everywhere you are using raw `new` to create a unique_ptr

e.g. `auto eventRequest = std::make_unique<PlayFabEmitEventRequest>();` it's shorter and prevents mismatching the types in the unique_ptr ctor and the `new` call